packages/now-cli/src/util/env/system-env.ts

```diff
@@ -0,0 +1,32 @@
+export const SYSTEM_ENV_VALUES = [
```

Comment:
Seems like it would be better to use an API endpoint instead of hard-coding these names so we could add more in the future.

Reply:
I agree, I created an internal ticket to track it: https://app.clubhouse.io/vercel/story/14475